client/src/Pages/Food/index.js

```diff
@@ -1,10 +1,101 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import IconLabeledButton from '../../components/Button';
+import ContainerComponent from '../../components/Container';
+import TextInputField from '../../components/InputFeild';
+import CardComponent from '../../components/Card';
+
+const axios = require('axios');
```

Comment:
Does it work to rewrite this as `import axios from 'axios'`? Then all of your file imports would be consistent.

Reply:
of course, this code is just for experimenting and learning how to use Axios